js/main.js

```diff
@@ -0,0 +1,63 @@
+var form = document.getElementById('form');
+form.addEventListener('submit', function(e) {
+
+  e.preventDefault();
+
+  var arrayForPage = [];
+
+  function FizzBuzz (a, b) {
+    this.lowNumber = a;
+    this.highNumber = b;
+  }
```

Comment:
You shouldn't declare constructors within event handlers; most of this code should be outside the event handler.